src/main/java/frc/robot/subsystems/CoralIntake.java

```diff
@@ -39,11 +39,12 @@ public class CoralIntake extends SubsystemBase {
   // Initialize arm SPARK. We will use MAXMotion position control for the arm, so we also need to
   // initialize the closed loop controller and encoder.
 
-  private enum CoralIntakeState {
+  public enum CoralIntakeState {
     STOW,
     HANDOFF,
     INTAKE,
-    EXTAKE
+    EXTAKE,
+    POOP_L1
   }
```

Comment:
these states should match the ones in the diagram i.e. the commands i.e. include the "ready" states

Reply:
if these are setpoints, then you should rename this and make a separate state enum

Reply:
and the commands should update the state instead of the setpoint command